src/components/Reports/FinancialAccountsReport/Header/TransactionsHeader.test.tsx

```diff
@@ -128,11 +127,12 @@ describe('Financial Account Header', () => {
     const { getByTestId } = render(<Components disableExportCSV={true} />);
 
     const view = getByTestId('FinancialAccountHeader');
-
     const searchInput = within(view).getByRole('textbox');
 
-    userEvent.type(searchInput, 'test');
+    await userEvent.type(searchInput, 'test');
```

Comment:
`userEvent.type` doesn't return a promise

```suggestion
    userEvent.type(searchInput, 'test');
```